### PR TITLE
CLOUDP-57846: List global alerts in mongocli

### PR DESCRIPTION
**Pull request by andreaangiolillo**
_Jira ticket:_ [CLOUDP-57846](https://jira.mongodb.org/browse/CLOUDP-57846)

## Checklist

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run `make fmt` and formatted my code
- [x] **I have tested my solution with my local OM**

## Further comments

`./bin/mongocli om alerts global list --profile OPS`

```json 
{
	"links": [
		{
			"rel": "self",
			"href": "http://localhost:9080/api/public/v1.0/globalAlerts?pageNum=1\u0026itemsPerPage=100"
		}
	],
	"results": [
		{
			"id": "5e7b354f2e676a3d207a845c",
			"groupId": "5e7b34822e676a3d207a8399",
			"alertConfigId": "5e7b35292e676a3d207a8431",
			"eventTypeName": "USERS_WITHOUT_MULTI_FACTOR_AUTH",
			"created": "2020-03-25T10:41:19Z",
			"updated": "2020-03-25T10:53:28Z",
			"status": "OPEN",
			"lastNotified": "2020-03-25T10:41:28Z",
			"acknowledgedUntil": "2020-11-01T03:59:28Z",
			"acknowledgementComment": "TEST",
			"acknowledgingUsername": "ydxjuumo",
			"links": [
				{
					"rel": "self",
					"href": "http://localhost:9080/api/public/v1.0/globalAlerts/5e7b354f2e676a3d207a845c"
				}
			]
		}
	],
	"totalCount": 1
}

### CODE REVIEW

File: internal/cli/ops_manager_alerts_global_list.go [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+}
+
+func (opts *opsManagerAlertsGlobalListOpts) init() error {
+	var err error
+	opts.store, err = store.New()
+	return err
+}
+
+func (opts *opsManagerAlertsGlobalListOpts) Run() error {
+	alertOpts := om.AlertsListOptions{
```
Comment (gssbzn) [inline internal/cli/ops_manager_alerts_global_list.go]: [q] what about pagination? 
[nit] you could also do `&om.AlertsListOptions{` here and not do the `&` on line 42

File: internal/store/global_alerts.go [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+}
+
+type GlobalAlertsStore interface {
+	GlobalAlertLister
+}
+
+// GlobalAlerts encapsulate the logic to manage different cloud providers
+func (s *Store) GlobalAlerts(opts *om.AlertsListOptions) (*om.GlobalAlerts, error) {
+	switch s.service {
+	case config.OpsManagerService, config.CloudManagerService:
```
Comment (gssbzn) [inline internal/store/global_alerts.go]: I think this is `config.OpsManagerService` only so you may need to remove `config.CloudManagerService`

File: internal/cli/ops_manager_alerts_global_list.go [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+func (opts *opsManagerAlertsGlobalListOpts) init() error {
+	var err error
+	opts.store, err = store.New()
+	return err
+}
+
+func (opts *opsManagerAlertsGlobalListOpts) Run() error {
+	alertOpts := &om.AlertsListOptions{
+		Status: opts.status,
+		ListOptions: mongodbatlas.ListOptions{
```
Comment (gssbzn) [inline internal/cli/ops_manager_alerts_global_list.go]: just as FYI (no need for any change here) you can also do
```go
alertOpts := &om.AlertsListOptions{}
alertOpts.PageNum = opts.pageNum
```
go has automatic delegation for embedded structs, this is just style and there's no advantage of one over the other, so just letting you know

Comment (gssbzn) [reply internal/cli/ops_manager_alerts_global_list.go]: Also for future work, we should do with pagination what we did with delete that we embed a struct with the common things that all paginated resources do

Comment (andreaangiolillo) [reply internal/cli/ops_manager_alerts_global_list.go]: I will create the activity on Jira

**Pull request merged**